services/openshift/scripts/openshift_provision

```diff
@@ -129,6 +129,7 @@ if [ ! -f ${ORIGIN_DIR}/configured.templates ]; then
 
     #EAP
     /opt/adb/openshift/templates/eap64-mysql-persistent-s2i.json
```

Comment:
@hferentschik sorry did not get you? Are you suggesting for a name change or something?
I have not verified whether eap64-mysql-persistent-s2i works. I guess once we have build ready we will ask relevant users to test it out.